ui-components/src/page_components/listing/ListingCard.stories.tsx

```diff
@@ -82,11 +97,11 @@ export const withStackedTable = () => {
 export const WithHeaders = () => {
   return (
     <ListingCard
-      imageCardProps={{ ...standardImageCardProps }}
+      imageCardProps={{ ...standardImageCardProps, href: undefined }}
       tableProps={{ ...standardTableProps }}
-      footerButtons={[{ text: "See Details", href: "see-details-link" }]}
+      footerButtons={[{ text: "See Details", href: "see-details-link", ariaHidden: true }]}
```

Comment:
Is this just to show how the prop can be used? I still see the button in storybook. But for just an example of usage this is great!

Reply:
On the Listings page, for better accessibility we want the button to show up to people visually, but hidden from screen readers because the link itself has already been announced. This example shows that in action.

Reply:
ohhhhh gotcha. thanks for clearing that up.

Reply:
sure thing!